site/en/guide/checkpoint.ipynb

```diff
@@ -53,7 +53,7 @@
       "source": [
         "\u003ctable class=\"tfo-notebook-buttons\" align=\"left\"\u003e\n",
         "  \u003ctd\u003e\n",
-        "    \u003ca target=\"_blank\" href=\"https://www.tensorflow.org/guide/checkpoint\"\u003e\u003cimg src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" /\u003eView on TensorFlow.org\u003c/a\u003e\n",
+        "    \u003ca target=\"_blank\" href=\"https://www.tensorflow.org/guide/checkpoints\"\u003e\u003cimg src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" /\u003eView on TensorFlow.org\u003c/a\u003e\n",
```

Comment:
Please revert this change. We're in the middle of switching the site around for TF2, the source of truth for these files is GitHub: https://github.com/tensorflow/docs/tree/master/site/en
And the site will follow shortly. Thanks

Reply:
I revert this change 
Thanks for information